api/valor_api/backend/models.py

```diff
@@ -99,7 +100,8 @@ class Prediction(Base):
         ForeignKey("annotation.id"), nullable=True
     )
     label_id: Mapped[int] = mapped_column(
-        ForeignKey("label.id"), nullable=False
+        ForeignKey("label.id"),
+        nullable=True,
```

Comment:
Please revert this changes as it does not affect your PR but may affect other evaluations. Since no labels are created, predictions will also never be created.